nixos/modules/services/desktops/system76-scheduler.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, ... }:
+
+let
+  cfg = config.services.system76-scheduler;
+
+  readDefaultConfig = path:
+    lib.fromRON (builtins.unsafeDiscardStringContext (builtins.readFile "${cfg.package}/data/${path}"));
```

Comment:
This is using import-from-derivation, which is generally not allowed in nixpkgs. Since the result here is used to build the manual, it means that for every manual build by everybody, the system76 package is downloaded/built and the RON Nix parser run on the config files, which is definitely not okay.

Defaults like these just can't be exposed in the manual.

Reply:
Ah you're right, I guess it only uses IFD if the module is enabled